playbooks/roles/certificates/tasks/ca-server.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: "Generate RSA keys for the CA and Server"
```

Comment:
How do you feel about removing the "RSA" bits from the `name`s in the certificates roles? One day I hope we can have `ECDSA` certificates as well :-) For the level of detail required for a `name` output maybe something like "Generate private keys for the CA and Server" would be sufficient and save updating in the future if we change away from RSA.